### DISCUSSION TITLE
Taiko circles don't load correctly in editor when rewinding back immediately after kiai starts

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#18472] [Q&A] [answered]
Let the editor play until it gets to the kiai, then seek back to right before the kiai. I can consistently reproduce this on any map with a kiai section. (Also happens with the osu! (triangles) skin; the circles don't fully load.)

(Logs just in case)
[updater.log](https://github.com/ppy/osu/files/8792653/updater.log)
[database.log](https://github.com/ppy/osu/files/8792654/database.log)
[network.log](https://github.com/ppy/osu/files/8792655/network.log)
[performance.log](https://github.com/ppy/osu/files/8792656/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8792657/runtime.log)

https://user-images.githubusercontent.com/62194990/170852559-06448835-6a0d-42c6-8659-00bc733965f4.mp4

**Comment by bdach** [comment, marked as answer]
#18478
